ZenPacks/zenoss/Microsoft/Windows/datasources/PerfmonDataSource.py

```diff
@@ -608,6 +607,15 @@ def _parse_deferred_result(self, result):
                 if stderr:
                     ps_error = ' '.join(stderr)
                     LOG.debug('stderr: {}'.format(ps_error))
+                    if "Attempting to perform the InitializeDefaultDrives operation on the 'FileSystem' provider failed." in ps_error:
```

Comment:
actually, this is a red herring and we don't need to report this as an event.  so if you do find this in stderr, then we need to ignore it as the perf data should still be there.  can you also add this check into the search_corrupt_counters method?

this error has nothing to do with actual collection, it's just something that gets written to stderr by powershell when running any script.